frontend/pages/ManageControlsPage/SetupExperience/cards/SetupAssistant/SetupAssistant.tests.tsx

```
@@ -49,11 +49,13 @@ describe("SetupAssistant", () => {
 
     await waitFor(() => {
       expect(
-        screen.getByText(/first turn on automatic enrollment/)
+        screen.getByText(/Additional configuration required/)
       ).toBeInTheDocument();
     });
     expect(
-      screen.getByText(/Add an automatic enrollment profile/)
+      screen.getByText(
+        /Turn on MDM and automatic enrollment to customize/
+      )
```

ESLint

[error] Replace `⏎········/Turn·on·MDM·and·automatic·enrollment·to·customize/⏎······` with `/Turn·on·MDM·and·automatic·enrollment·to·customize/`

(prettier/prettier)